check_hudi_status.py

```
@@ -0,0 +1,18 @@
+from pyspark.sql import SparkSession
+
+spark = SparkSession.builder.appName("check-hudi-status").getOrCreate()
+
+try:
+    df = spark.read.format("hudi").load("/tmp/hudi_data/sec_bhavdata")
```

Ruff (0.14.10)

Probable insecure usage of temporary file or directory: "/tmp/hudi_data/sec_bhavdata"

(S108)